BigBatch/Views/BatchedVC1.swift

```diff
@@ -263,87 +264,43 @@ class BatchedVC1: UIViewController, UITableViewDelegate, UITableViewDataSource,
-        let stringToNumberOzAmount9 = Double(batchedOzAmount9) ?? 0.0
-        let stringToNumberOzAmount10 = Double(batchedOzAmount10) ?? 0.0
-        let stringToNumberOzAmount11 = Double(batchedOzAmount11) ?? 0.0
-        let stringToNumberOzAmount12 = Double(batchedOzAmount12) ?? 0.0
-        let stringToNumberOzAmount13 = Double(batchedOzAmount13) ?? 0.0
-        let stringToNumberOzAmount14 = Double(batchedOzAmount14) ?? 0.0
-        let stringToNumberOzAmount15 = Double(batchedOzAmount15) ?? 0.0
-        let stringToNumberOzAmount16 = Double(batchedOzAmount16) ?? 0.0
-        
-        
-        mlAmount1 = stringToNumberOzAmount1 * 29.5735
-        mlAmount2 = stringToNumberOzAmount2 * 29.5735
-        mlAmount3 = stringToNumberOzAmount3 * 29.5735
-        mlAmount4 = stringToNumberOzAmount4 * 29.5735
-        mlAmount5 = stringToNumberOzAmount5 * 29.5735
-        mlAmount6 = stringToNumberOzAmount6 * 29.5735
-        mlAmount7 = stringToNumberOzAmount7 * 29.5735
-        mlAmount8 = stringToNumberOzAmount8 * 29.5735
-        mlAmount9 = stringToNumberOzAmount9 * 29.5735
-        mlAmount10 = stringToNumberOzAmount10 * 29.5735
-        mlAmount11 = stringToNumberOzAmount11 * 29.5735
-        mlAmount12 = stringToNumberOzAmount12 * 29.5735
-        mlAmount13 = stringToNumberOzAmount13 * 29.5735
-        mlAmount14 = stringToNumberOzAmount14 * 29.5735
-        mlAmount15 = stringToNumberOzAmount15 * 29.5735
-        mlAmount16 = stringToNumberOzAmount16 * 29.5735
-        let mlAmountArray = [mlAmount1, mlAmount2, mlAmount3, mlAmount4, mlAmount5, mlAmount6, mlAmount7, mlAmount8, mlAmount9, mlAmount10, mlAmount11, mlAmount12, mlAmount13, mlAmount14, mlAmount15, mlAmount16]
+  
+        mlAmount1 = (Double(batchedOzAmount1) ?? 0.0) * 29.5735
+        mlAmount2 = (Double(batchedOzAmount2) ?? 0.0) * 29.5735
+        mlAmount3 = (Double(batchedOzAmount3) ?? 0.0) * 29.5735
+        mlAmount4 = (Double(batchedOzAmount4) ?? 0.0) * 29.5735
+        mlAmount5 = (Double(batchedOzAmount5) ?? 0.0) * 29.5735
+        mlAmount6 = (Double(batchedOzAmount6) ?? 0.0) * 29.5735
+        mlAmount7 = (Double(batchedOzAmount7) ?? 0.0) * 29.5735
+        mlAmount8 = (Double(batchedOzAmount8) ?? 0.0) * 29.5735
+        mlAmount9 = (Double(batchedOzAmount9) ?? 0.0) * 29.5735
+        mlAmount10 = (Double(batchedOzAmount10) ?? 0.0) * 29.5735
+        mlAmount11 = (Double(batchedOzAmount11) ?? 0.0) * 29.5735
+        mlAmount12 = (Double(batchedOzAmount12) ?? 0.0) * 29.5735
+        mlAmount13 = (Double(batchedOzAmount13) ?? 0.0) * 29.5735
+        mlAmount14 = (Double(batchedOzAmount14) ?? 0.0) * 29.5735
+        mlAmount15 = (Double(batchedOzAmount15) ?? 0.0) * 29.5735
+        mlAmount16 = (Double(batchedOzAmount16) ?? 0.0) * 29.5735
+        
+        var mlAmountArray = [mlAmount1, mlAmount2, mlAmount3, mlAmount4, mlAmount5, mlAmount6, mlAmount7, mlAmount8, mlAmount9, mlAmount10, mlAmount11, mlAmount12, mlAmount13, mlAmount14, mlAmount15, mlAmount16]
```

Comment:
Try put this in a for loop. 

For extra points you could consolidate all those batchedOzAmount1...batchedOzAmount16 variables into a dictionary and then run a map function over it to return the array you need, but you might need to keep it in this format depending on how your passing the data around the app